upgrade/UPGRADE-v9.1.0-dev.md

```diff
@@ -68,3 +68,10 @@ There you can find links to upgrade notes for other versions too.
 
 - fix displaying '+' sign in product filter ([#2023](https://github.com/shopsys/shopsys/pull/2023))
     - see #project-base-diff to update your project
+
+- added more coding standards ([#2035](https://github.com/shopsys/shopsys/pull/2035))
+    - disallowed usage of `empty()` is one which must be fixed manually
+        - first of all you should keep in ming the `empty` ignores undefined offsets and this state should be probably kept
+        - if `empty` is used for checking elements of array it can be replaced via `count($array)`
+        - if `empty` is used for checking for empty values it should be replaced via `in_array($value, [null, false, '', '0', 0, 0.0, []])`
```

Comment:
in that case, I just simulate the empty behavior

Reply:
the whole line was removed and replaced with better instructions